docs/UPTIME_STATUS_PAGE.md

```diff
@@ -0,0 +1,22 @@
+# GigaDB server monitoring tool: UptimeRobot
+
+UptimeRobot provides uptime monitoring service, its free tier plan allows up to 50 monitors with minimum monitoring interval `5mins` and 1 status page.
+Currently, the GigaDB monitoring page is at [here](https://stats.uptimerobot.com/LGVQXSkN1y).
+
+### How GigaDB servers are monitored
+
+| Server name | Type of Check | value to check | 
+| --- | --- | --- |
+| GigaDB FTP server | HTTP(S) | https://ftp.cngb.org/pub/gigadb/pub/10.5524/ |
+| GigaDB Web Site (LIVE) | HTTP(S) | https://gigadb.org |
+| GigaDB Database Server (LIVE) | Port | database-live.gigadb.org:5432 |
```

Comment:
I guess we can create a CNAME DNS record for `gigadb.org` so that RDS endpoints maps to `database-live.gigadb.org` and `database-staging.gigadb.org`. If we are careful then the RDS endpoint should not change if we keep a consistent instance identifier for staging and live RDS instances - see [here](https://forums.aws.amazon.com/thread.jspa?messageID=486170#486199).